ScreenshotTests/SnapshotTests.swift

```diff
@@ -7,7 +7,7 @@ import XCTest
 class SnapshotTests: BaseTestCaseL10n {
 
     func test01FirstRunScreens() {
-        waitForExistence(app.buttons["IntroViewController.startBrowsingButton"], timeout: 10)
+        waitForExistence(app.buttons["Get Started"], timeout: 10)
```

Comment:
Should we use a `launchArguments` from `BaseTestCaseL10n` to simulate the browser is opened for the very first time so that the welcome screen shows up?

Reply:
Since these are tests not to find issues but to get screenshots, it is fine to have this and run them in a particular order... 
If you see the regular test suite, there is a `[disableFirstRun](https://github.com/mozilla-mobile/focus-ios/blob/51d3c0d879048cdce43fc14cbcca9ebc71686c58/XCUITest/BaseTestCase.swift#L13)` to manage that when running the tests

Reply:
Is "Get Started" an ID or the text label? we need to use IDs, since the label will change as the test runs in a different locale than English

Reply:
Looks like that's a text so that would not work with these tests. We also need to go to the second screen and take a screenshot of that one too. 
We have these two screens:

![Simulator Screen Shot - iPhone 14 Pro - 2023-02-09 at 17 35 52](https://user-images.githubusercontent.com/1897507/217879938-70444232-ddf2-420f-9e66-c0745123a1bb.png)
![Simulator Screen Shot - iPhone 14 Pro - 2023-02-09 at 17 35 55](https://user-images.githubusercontent.com/1897507/217879953-c9d115bf-7991-4f58-99e0-2a236d9482e2.png)

So the test would be something like:
```
func test01FirstRunScreens() {
        waitForExistence(app.collectionViews.cells.images["icon_background"], timeout: 10)
        snapshot("00FirstRun")
        app.collectionViews.cells.images["icon_background"].swipeLeft()
        waitForExistence(app.collectionViews.cells.images["icon_hugging_focus"], timeout: 3)
        snapshot("01FirstRun")
    }
```